modules/microshift-greenboot-check-boot.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-check-boot_{context}"]
+= Checking an update's success with greenboot
+
+After an update, you can determine whether the update was declared valid by checking the boot. Such a health check can help you determine if you the system is working properly and you can proceed without a rollback.
+
+The health check script for updates is installed into the `/etc/greenboot/check/required.d` directory, and is automatically executed during the system boot. Exiting the script with a non-zero status will have the boot declared as failed.
```

Comment:
It's not for updates. It runs on every boot regardless of whether an update happened or not.

Reply:
I was thinking of updates as a special use-case to call out. No? Let's looks at the docs together. I sent you a meeting invite.